app/assets/javascripts/osom-tables.js

```diff
@@ -150,8 +150,8 @@
 
   // Callback for successful ajax load of table data
   function table_load_success(container, new_content, url) {
-    var new_container = $('<div class="osom-table ">').append($(new_content).filter('div.osom-table'));
-    container.html(new_container);
+    var new_container = $(new_content);
+    container.html($(new_content).children());
```

Comment:
just 1 line `var new_container = container.replaceWith(new_content);`

Reply:
@dangquangthai I think `replaceWith` produce different result to `html`. In this PR , I intend to revert to what stable before the #13 , any refactor or code optimize is welcome in another PR, IMHO. 👍